QuickOSM/test/test_connexion_nominatim.py

```diff
@@ -0,0 +1,88 @@
+"""Tests for Nominatim API requests."""
+
+from qgis.testing import unittest
+
+from QuickOSM.core.api.nominatim import Nominatim
+from QuickOSM.core.exceptions import (
+    NominatimAreaException,
+    NominatimBadRequest,
+)
+from QuickOSM.definitions.nominatim import NOMINATIM_SERVERS
+
+__copyright__ = 'Copyright 2019, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+class MyTestCase(unittest.TestCase):
+    def test_wrong_request(self):
+        """Test wrong request.
+
+        This is test is using internet.
+        """
+        nominatim = Nominatim(NOMINATIM_SERVERS[0])
+
+        with self.assertRaises(NominatimBadRequest) as e:
+            nominatim.query('')
+
+        self.assertEqual(str(e.exception), 'Nominatim hasn\'t found any data for an area called "".')
+
+    def test_wrong_request_area(self):
+        """Test wrong osm_id request.
+
+        This is test is using internet.
+        """
+        nominatim = Nominatim(NOMINATIM_SERVERS[0])
+
+        with self.assertRaises(NominatimAreaException) as e:
+            nominatim.get_first_polygon_from_query('CosWos')
+
+        self.assertEqual(str(e.exception),
+                         '(\'No named area found for OSM relation called "CosWos".\', '
+                         '\'No OSM polygon (relation) has been found, you should try the '
+                         '"Around" query which will search for other OSM type.\')')
```

Comment:
Minor, but we often pull left as much as we can.

The indentation should be a multiple of 4 spaces.
L41, it might not be a multiple of 4.

"self.assertEqual(" then press "Enter".